package.json

```
@@ -2180,6 +2180,11 @@
 					"group": "inline@2",
 					"when": "view =~ /(pr|prStatus):github/ && viewItem =~ /description/"
 				},
+				{
+					"command": "pr.openPullRequestOnGitHub",
+					"group": "inline@3",
+					"when": "view =~ /(pr|prStatus):github/  && viewItem =~ /description/ && github:activePRCount >= 2"
```

One more thing I didn't notice before: I think this should only show in the prStatus view:

```suggestion
					"when": "view == prStatus:github && viewItem =~ /description/ && github:activePRCount >= 2"
```